.github/workflows/publish_release.yml

```diff
@@ -0,0 +1,77 @@
+name: Publish a new release
```

Comment:
Two different workflows: one for manual releases, and an other for automatic releases

Reply:
Done